src/js/react/apps/maternity-and-child-health-clinic-search/types/MaternityAndChildHealthClinic.ts

```diff
@@ -0,0 +1,23 @@
+type ImageOverride = {
```

Comment:
This ImageOverride type could be added to `/public/themes/contrib/hdbt/src/js/types` common types folder because it's used in three searches: this, health stations and schools. You could also update health station and school search to use the common type after you have added it.